### ISSUE TITLE
Group avatar

### ISSUE THREAD

**Issue by jflefebvre06** [gitlab4j/gitlab4j-api#533]
Could you please add avatar (introduced GitLab 12.9) into group api

**Comment by gmessner** [comment]
@jflefebvre06 I will add a method using the same pattern as projects and users:
```
public Group setGroupAvatar(Object groupIdOrPath, File avatarFile)
```
Should have a release out in the next day or two.

**Comment by jflefebvre06** [comment]
Could you please add 2 methods, one with file and other with input stream ?

**Comment by gmessner** [comment]
@jflefebvre06 
Unfortunately, a lot more work would need to be done to support an input stream, this library does not currently do that at all, and I currently do not have the time to implement that just to support this one use.

**Comment by gmessner** [comment]
@jflefebvre06 Actually, it ends up being quite a bit of work to support an InputStream for the avatar upload.  I do not have enough time to do this amount of work for this one use, so will release with just the one method.

**Comment by jflefebvre06** [comment]
Ok thanks

**Comment by gmessner** [comment]
@jflefebvre06 Just released 4.14.21 with the one method to set the group avatar, sorry for not having time to get the steam one done.

**Comment by gmessner** [comment]
@jflefebvre06 Since this has been released I am closing this issue.  If you have problems with the implementation please re-open this ticket.
